src/os/bluestore/BlueStore.cc

```diff
@@ -6282,6 +6282,7 @@ void BlueStore::_txc_state_proc(TransContext *txc)
     case TransContext::STATE_PREPARE:
       txc->log_state_latency(logger, l_bluestore_state_prepare_lat);
       if (txc->ioc.has_pending_aios()) {
+	txc->no_rtc = true;
```

Comment:
The meaning of no_rtc is confusing.  Making it a positive (can_rtc or can_submit_txn_sync) or describe what it actually means (is_pipelined_io?) would be better.